### PR TITLE
update to truffle 0.5 and web3 1.0

### PR DESCRIPTION
**Pull request by rotcivegaf**

### CODE REVIEW

File: contracts/libs/SafeERC20.sol [hunk 1]
```diff
@@ -92,9 +92,9 @@ library SafeERC20 {
    * and the new allowance by unfortunate transaction ordering. One possible solution to mitigate this
    * race condition is to first reduce the spender's allowance to 0 and set the desired value afterwards:
    * https://github.com/ethereum/EIPs/issues/20#issuecomment-263524729
-   * 
+   *
    * @param _token erc20 The address of the ERC20 contract
-   * @param _spender The address which will spend the funds.
+   * @param _spender The asafeTransferFromddress which will spend the funds.
```
Comment (nachomazzara) [inline contracts/libs/SafeERC20.sol]: Typo here. I think that we should keep `address` right?

File: package.json [hunk 2]
```diff
@@ -40,11 +39,11 @@
...
     "truffle-hdwallet-provider": "0.0.3",
-    "web3": "^0.20.4"
+    "web3": "^0.20.7"
   },
   "dependencies": {
-    "openzeppelin-eth": "^2.0.2",
-    "zos": "^2.0.0",
-    "zos-lib": "^2.0.0"
+    "openzeppelin-eth": "^2.1.3",
+    "zos": "^2.2.2",
```
Comment (nachomazzara) [inline package.json]: I think that some libraries here are useless. Do you mind checking it?

**Review by nachomazzara — COMMENTED** [review]
:fire: Thank you so much for the effort!! 

I will run the test later and marge it if everything is ok

File: contracts/libs/SafeERC20.sol (hunk 1 above)
Comment (rotcivegaf) [reply contracts/libs/SafeERC20.sol]: Sorry, i used replace all

File: package.json (hunk 2 above)
Comment (rotcivegaf) [reply package.json]: Done

**Comment by nachomazzara** [comment]
One more little thing: Could you install `npm i truffle` and add at `truffle-config.json` 

```
development: {
      gas: 6721975,
      host: 'localhost',
      port: 8545,
      network_id: '*' // eslint-disable-line camelcase
    },
```

Test won't run if not

**Review by nachomazzara — APPROVED** [review]

**Pull request merged**